pkg/blob/nodeserver.go

```diff
@@ -575,3 +582,20 @@ func (d *Driver) ensureMountPoint(target string, perm os.FileMode) (bool, error)
 	}
 	return !notMnt, nil
 }
+
+func waitForMount(path string, intervel, timeout time.Duration) error {
```

Comment:
pls add a ut for this func, e.g. if `bind mount` to local disk, it should return `notMount as true`

Reply:
and also add some logging to check when the blobfuse mount would ready after a few retries.

Reply:
added.